sources/packages/web/src/components/common/ConfirmEnrolment.vue

```diff
@@ -0,0 +1,84 @@
+<template>
+  <check-permission-role :role="Role.StudentConfirmEnrolment">
+    <template #="{ notAllowed }">
+      <v-btn
+        v-if="isConfirmCOEEnabled"
+        variant="text"
+        color="primary"
+        @click="submitConfirmEnrolment"
+        :disabled="notAllowed"
+        ><span class="text-decoration-underline"
+          ><strong>Confirm Enrolment</strong></span
```

Comment:
for bold pls use the style class that uses `$font-bold;` added by @hellolynn-tbtb

Reply:
This one ? 
![image](https://user-images.githubusercontent.com/54600590/211111781-9bd1c97c-f35a-41c6-9dda-4c975688ba2f.png)

Reply:
I was trying to create a class which just sets the font-family to $font-bold and $font-weight to $font-weight-bold.

The name of the above class was very accurate for that purpose and atomic. I recommend that this class `bold-text` must have been more atomic excluding color definitely and not sure about other properties. Also the variable $font-bold may have been `$font-family-bold` to be more specific.

Coming back to this PR, I created an atomic class `app-font-bold` to set font-weight and font-font family for bold texts.

![image](https://user-images.githubusercontent.com/54600590/211390042-06ed23e6-3672-4f0e-b305-fe21b86a0ee4.png)

Reply:
Created class font-bold to use for bold only purpose. 

![image](https://user-images.githubusercontent.com/54600590/211400481-9b62c580-e438-442e-838a-df9a8f5f8aa8.png)